bootstrap/004_labels.sql

```diff
@@ -107,6 +122,12 @@ SELECT *,$$;
         s := s || $$ else 'Other' end as "$$ || group_name || '", ';
     end if;
 
+    let dynamic_groups cursor for
+        select group_name, condition from internal.labels
+        where is_dynamic;
+    for dgrp in dynamic_groups do
+        s := s || dgrp.condition || ' as "' || dgrp.group_name || '",';
```

Comment:
I would add the `if label_condition is not null label_condition else 'Other' AS group_name` here so we always have an 'other' bucket

Reply:
Makes sense. I'll add 'other' bucket to the group.